data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/chance_to_hit/chance_to_hit_add_sub.cfg

```diff
@@ -0,0 +1,40 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [chance_to_hit]value=
+##
+# Actions:
+# Give the leaders a chance_to_hit ability with both the value, add, and sub attributes, adding up to 10
```

Comment:
... but effectively zero because sub completely overrides the add attribute.

Reply:
Updated.